src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -637,6 +637,17 @@
   <data name="CommandPaletteMenuItem" xml:space="preserve">
     <value>Command Palette</value>
   </data>
+  <data name="AppName" xml:space="preserve">
+    <value>Windows Terminal</value>

```

Comment:
We seriously don't have an existing resource with "Windows Terminal" already?

Reply:
Yes in fact we do! 😄 Going to manually pull it from CascadiaPackage's resources.

Reply:
wait uh, if you're gonna pull it from the `CascadiaPackage` resources, do we need _this_ string still?